codegen/src/main/java/org/web3j/codegen/unit/gen/Parser.java

```diff
@@ -20,6 +20,7 @@
 import java.util.List;
 
 import org.junit.jupiter.api.Assertions;
+import sun.reflect.generics.reflectiveObjects.ParameterizedTypeImpl;
```

Comment:
Do we really want to use something under the sun package? It's not recommended, but be an alternative..?

Reply:
Fixed. It should be ParameterizedType.